dsperse/src/backends/jstprove.py

```
@@ -287,27 +346,33 @@ def compile_circuit(
+        model = _add_zero_bias_to_conv(model)
+
+        fd, preprocessed_path = tempfile.mkstemp(suffix=".onnx")
+        os.close(fd)
         try:
+            onnx.save(model, preprocessed_path)
             self._run_command("compile", [
-                "-m", str(model_path),
+                "-m", preprocessed_path,
                 "-c", str(circuit_path),
             ])
             return True, None
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)